load_test/config/runtime.exs

```diff
@@ -7,6 +7,7 @@ config :logger, :console,
 config :load_test, port: String.to_integer(System.get_env("PORT") || "2999")
 config :load_test, nb_user: String.to_integer(System.get_env("NB_USER") || "1")
 
+config :load_test, sse_user_agent: System.get_env("SSE_USER_AGENT") || "neurow_load_test/1.0"
```

Comment:
Nit: Maybe replace the version number with the git commit hash ? (If we really need to know which version is actually running)